### PR TITLE
Added atom plugin link to usage.html

### PR DESCRIPTION
**Pull request by gliwka**
Added a reference to the atom-zeal plugin for the atom editor (Disclaimer: I'm the author of this plugin)

### CODE REVIEW

**Comment by trollixx** [comment]
Hi,

Thanks! I am not an Atom user myself but it is getting some level of popularity as far as I know. A few cosmetic changes would be nice before merging the change. Could you please squash both commits and change the message to `Usage: Add Atom plugin link`?

File: usage.html [hunk 1]
```diff
@@ -64,7 +64,7 @@ <h2 id="plugins">IDE Plugins</h2>
   <p>
     There is a Vim plugin available at <a href="https://github.com/KabbAmine/zeavim.vim">https://github.com/KabbAmine/zeavim.vim</a>.
   </p>
-
+
```
Comment (trollixx) [inline usage.html]: Please remove this white space, but keep the empty line.

File: usage.html [hunk 2]
```diff
@@ -73,4 +73,11 @@ <h2 id="plugins">IDE Plugins</h2>
 
   <p>A simpler way to integrate with Vim is a simple mapping, like</p>
   <pre><code>:nnoremap gz :!zeal --query "&lt;cword&gt;"&amp;&lt;CR&gt;&lt;CR&gt;</code></pre>
+  
+  
+  <h3>Atom</h3>
+    
+  <p>
+    There is a Atom plugin available at <a href="https://github.com/gliwka/atom-zeal">https://github.com/gliwka/atom-zeal</a>.
```
Comment (trollixx) [inline usage.html]: It should be `an Atom plugin`.

**Comment by gliwka** [comment]
Done. Squashed the commits and fixed the white spaces.

**Pull request merged**